site/assets/js/application.js

```diff
@@ -144,13 +161,23 @@
   })
 
   clipboard.on('success', function (event) {
+    var iconFirstChild = event.trigger.querySelector('.bi').firstChild
     var tooltipBtn = bootstrap.Tooltip.getInstance(event.trigger)
+    var namespace = 'http://www.w3.org/1999/xlink'
+    var originalXhref = iconFirstChild.getAttributeNS(namespace, 'href')
+    var originalTitle = event.trigger.title
 
     tooltipBtn.setContent({ '.tooltip-inner': 'Copied!' })
     event.trigger.addEventListener('hidden.bs.tooltip', function () {
       tooltipBtn.setContent({ '.tooltip-inner': btnTitle })
     }, { once: true })
     event.clearSelection()
+    iconFirstChild.setAttributeNS(namespace, 'href', originalXhref.replace('clipboard', 'check2'))
+
+    setTimeout(function () {
+      iconFirstChild.setAttributeNS(namespace, 'href', originalXhref)
```

Comment:
`setTimeout` maybe could be avoided, if we put the code-block on line 172 (inside hidden tooltip event)

Reply:
Thanks, just tried that. While it works, I prefer the not-so-instant change of the icon and tooltip. It feels a little jarring without the timeout, so I'm going to leave it as-is for now :).